charts/observability-pipelines-worker/templates/_helpers.tpl

```diff
@@ -47,6 +47,14 @@ Return the configuration key Secret name to be used based on provided values.
 {{- default $fullName .Values.datadog.configKeyExistingSecret | quote -}}
 {{- end -}}
 
+{{/*
+Return the pipeline ID Secret name to be used based on provided values.
```

Comment:
This replaces the `opw.configKeySecretName` partial above, correct? We should remove it if it's now unused.

Reply:
I think this is still necessary for backwards compatibility unless I'm misunderstanding something here.

Reply:
Oops, yeah - I temporarily lost my mind and forgot about the backward compatibility here.